src/lib/Components/Inbox/ActiveBids/index.tsx

```diff
@@ -90,7 +90,7 @@ export default createRefetchContainer(
       }
     }
   `
-)
+) as React.ComponentClass<Props>
```

Comment:
Why is this necessary here? Isn't this info encoded in the `class ActiveBids extends React.Component<Props, State>` signature?

Reply:
Correct. I need to think a bit on how to fix this, but this is imo a bug with the Relay typings. It specifies that `create*Container` returns a union of `React.ComponentClass` and `React.StatelessComponent`, the latter has no `ref` prop which the user of this component needs.